src/Module/ReallySimpleDiscovery.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Core\Renderer;
+
+/**
+ * Prints the rsd.xml
+ * @see http://danielberlinger.github.io/rsd/
+ */
+class ReallySimpleDiscovery extends BaseModule
+{
+	public static function rawContent()
+	{
+		header ('Content-Type: text/xml');
+		$tpl = Renderer::getMarkupTemplate('rsd.tpl');
```

Comment:
It's a little bit late, but I would prefer having all XML templates replaced with creating XML via the XML routines.

Reply:
There's a XML routine :X .. Sorry I didn't notice yet

Reply:
Yeah, that routine can be feed with an array that is then transformed in an XML. Have a look at how ```XML::fromArray``` is used in the current code.